app/views/whitehall_migration/show.html.erb

```diff
@@ -1,14 +1,49 @@
-<h1>Whitehall Migration: <%= @whitehall_migration.id %> </h1>
-<ul>
-    <li>Time started: <%= @whitehall_migration.created_at.strftime("At %T on %d/%m/%Y") %></li>
-    <li>Time finished: <%= @whitehall_migration.finished_at.nil? ? "Unfinished" : @whitehall_migration.finished_at %></li>
-    <li>Number of documents: <%= @whitehall_migration.document_imports.count %></li>
-</ul>
-
-<h3>Document Import Status</h3>
-
-<ul>
-    <% @whitehall_migration.document_imports.states.each_value do |state| %>
-        <li><%= "#{state}:" %> <%= @whitehall_migration.document_imports.where(state: "#{state}").count%></li>
-    <% end %>
-</ul>
+
+<% content_for :title, "Whitehall Migration: #{@whitehall_migration.id}" %>
+
+<%= render "govuk_publishing_components/components/summary_list", {
+  items: [
+    {
+      field: "Time started",
+      value: format_time_on_date(@whitehall_migration.created_at)
+    },
+    {
+      field: "Time finished",
+      value: @whitehall_migration.finished_at.nil? ? "unfinished" : @whitehall_migration.finished_at
+    },
+    {
+      field: "Number of documents",
+      value: @whitehall_migration.document_imports.count
+    }
+  ],
+  borderless: true
+} %>
+
+<h2 class="govuk-heading-m">Document import statuses</h2>
+<%
+  items = @whitehall_migration.document_imports.states.map do |state, _|
+    {
+      link: {
+        text: "#{state.titleize}: #{@whitehall_migration.document_imports.where(state: state).count}",
+        path: whitehall_migration_documents_path(migration_id: @whitehall_migration, state: state),
+      },
+    }
+  end
+%>
+
+<%= render "govuk_publishing_components/components/document_list", items: items %>
+
+<% random_items = @whitehall_migration.document_imports.completed.order(Arel.sql("RANDOM()")).first(10).map do |document|
+    {
+      link: {
+        text: document.document.current_edition.title
+      },
+      subtext: "View: <a href=\"#{Plek.find('whitehall-admin')}/government/admin/by-content-id/#{document.content_id}\">Whitehall</a> View: <a href=\"#{Plek.find('content-publisher')}/documents/#{document.content_id}:en\">Content Publisher</a>".html_safe
+    }
+  end
```

Comment:
I feel like we should have a partial that we can pass an array of document_imports into renders the document_list component. This would avoid the need for repetition here and on the documents view

Reply:
I am not sure if I understand... the `document_list` needs to take `items`. The `items` are constructed from some form of `document_imports` but they all vary a lot in how the values within `items` are generated... so I don't think one partial would fit all very easily. The `render "govuk_publishing_components/components/document_list"` already feels like we are using a partial anyway?